content/sensu-go/5.0/getting-started/sample-app.md

```diff
@@ -0,0 +1,363 @@
+
+sensuctl config set-namespace demo
+{{< /highlight >}}
+
+**3. Create `dev` user role with full-access to the `demo` namespace.**
+
+{{< highlight shell >}}
+sensuctl role create dev \
+--verb get,list,create,update,delete \
+--resource \* --namespace demo
+{{< /highlight >}}
+
+**4. Create `dev` role binding for `dev` group.**
+
+{{< highlight shell >}}
+sensuctl role-binding create dev --role dev --group dev
+{{< /highlight >}}
+
+**5. Create `demo` user that is a member of the `dev` group.**
+
+{{< highlight shell >}}
+sensuctl user create demo --interactive
+? Username: demo
+? Password: ********
+? Groups: dev
+{{< /highlight >}}
+
+**6. Reconfigure sensuctl to use the `demo` user and `demo` namespace.**
+
+{{< highlight shell >}}
+sensuctl configure
+? Sensu Backend URL: http://sensu.local
+? Username: demo
+? Password: ********
+? Namespace: demo
+? Preferred output format: tabular
+{{< /highlight >}}
+
+**7. Log in to the Sensu dashboard.**
+
+At http://webui.sensu.local/signin, sign in with your sensuctl username (`demo`) and password.
+
+## Deploying Sensu agents and InfluxDB
+
+**1. Deploy InfluxDB with a Sensu Agent sidecar**
+
+Create a Kubernetes ConfigMap for InfluxDB configuration
+
+{{< highlight shell >}}
+kubectl create configmap influxdb-config --from-file go/configmaps/influxdb.conf
+{{< /highlight >}}
+
+Deploy InfluxDB with a Sensu Agent sidecar
+
+{{< highlight shell >}}
+kubectl create -f go/deploy/influxdb.sensu.yaml
+{{< /highlight >}}
+
+**2. Create a Sensu pipeline to store metrics with InfluxDB.**
+
+Register a Sensu asset for the [Sensu InfluxDB handler][], and create an `influxdb` event handler.
+
+{{< highlight shell >}}
+sensuctl create -f go/config/assets/influxdb-handler.yaml
+
+sensuctl create -f go/config/handlers/influxdb.yaml
+{{< /highlight >}}
+
+**3. Deploy Sensu agent sidecars for the dummy app instances.**
+
+{{< highlight shell >}}
+kubectl apply -f go/deploy/dummy.sensu.yaml
+{{< /highlight >}}
+
+## Monitoring an app
+
+Let's take a look at what we're monitoring.
+
+{{< highlight shell >}}
+sensuctl entity list
+{{< /highlight >}}
+
+We can see the Sensu agents we have installed on the dummp app, InfluxDB, and Grafana pods with their last seen timestamp.
+
+{{< highlight shell >}}
+ID               Class    OS                   Subscriptions                           Last Seen            
+─────────────────────────── ─────── ─────── ─────────────────────────────────────────── ─────────────────────────────── 
+dummy-6c57b8f868-ft5dz      agent   linux   dummy,entity:dummy-6c57b8f868-ft5dz         2018-11-20 18:43:15 -0800 PST  
+dummy-6c57b8f868-m24hw      agent   linux   dummy,entity:dummy-6c57b8f868-m24hw         2018-11-20 18:43:15 -0800 PST  
+grafana-5b88f8df8d-vgjtm    agent   linux   grafana,entity:grafana-5b88f8df8d-vgjtm     2018-11-20 18:43:14 -0800 PST  
+influxdb-78d64bcfd9-8km56   agent   linux   influxdb,entity:influxdb-78d64bcfd9-8km56   2018-11-20 18:43:12 -0800 PST  
```

Comment:
In going through the instructions on the demo project, I'm not seeing a Grafana entity. This seems like we might be missing something here.